zuko/flows/polynomial.py

```diff
@@ -87,13 +87,13 @@ def __init__(
         features: int,
         context: int = 0,
         degree: int = 16,
-        linear: bool = False,
+        bound: float = 10.0,
```

Comment:
Providing the `bound` option goes against the current convention to not provide transformation options in the flow constructors. Users are expected to build their own flows if they want more control. I am happy to reconsider this though, as it gives more flexibility, but not in the current PR as it would require changes in other places in the code base.

Reply:
ok, i'll revert it.